plugins/chat/assets/javascripts/discourse/components/chat-channel-card.hbs

```diff
@@ -11,7 +11,7 @@
     <div class="chat-channel-card__header">
       <LinkTo
         @route="chat.channel"
-        @models={{array @channel.id (slugify-channel @channel)}}
+        @models={{array (slugify-channel @channel) @channel.id}}
```

Comment:
Im thinking, maybe part of this we introduce a `@channel.slug` so we don't have to call `slugify-channel` everywhere. Given we have  slug in the backend now, this should be easy to do.